doc/conf.py

```diff
@@ -43,8 +46,16 @@
     "pylint_options",
     "sphinx.ext.autosectionlabel",
     "sphinx.ext.intersphinx",
+    "sphinx_reredirects",
```

Comment:
This project doesn't seem very well maintained. After the initial release of `0.0.1` a lot of open issue were never dealt with.

Are we sure want to use it?

Reply:
Yeah 0.0.1 does not inspire confidence. But the current version is working well enough, and it's just a dict of simple redirect so worst case scenario we can still switch later if required.

Reply:
( https://pylint--6613.org.readthedocs.build/en/6613/intro.html / https://pylint--6613.org.readthedocs.build/en/6613/support.html )